style.css

```diff
@@ -1,2 +1,199 @@
-body { font-family: "Montserrat", serif; }
+body { 
+  background: #FFF;
+  font-family: "Montserrat", serif;
+  height: 2700px;
+  width: 1440px;
+}
 
+.wordmarksmall {
+  /*height: 51px;*/
+ /* margin-right: 1231px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/IFW/pullRequest?prid=6044074)